packages/ckeditor5-ui/src/dialog/dialogview.ts

```diff
@@ -0,0 +1,699 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/dialog/dialogview
+ */
+
+import {
+	type EventInfo,
+	type Locale,
+	type DecoratedMethodEvent,
+	KeystrokeHandler,
+	FocusTracker,
+	Rect,
+	global,
+	toUnit
+} from '@ckeditor/ckeditor5-utils';
+import ViewCollection from '../viewcollection.js';
+import View from '../view.js';
+import FormHeaderView from '../formheader/formheaderview.js';
+import ButtonView from '../button/buttonview.js';
+import { type ButtonExecuteEvent } from '../button/button.js';
+import FocusCycler, { isViewWithFocusCycler,
+	type FocusCyclerBackwardCycleEvent,
+	type FocusCyclerForwardCycleEvent
+}
+	from '../focuscycler.js';
+import DraggableViewMixin, { type DraggableView, type DraggableViewDragEvent } from '../bindings/draggableviewmixin.js';
+import DialogActionsView, { type DialogActionButtonDefinition } from './dialogactionsview.js';
+import DialogContentView from './dialogcontentview.js';
+import type EditorUI from '../editorui/editorui.js';
+
+import '../../theme/components/dialog/dialog.css';
+// @if CK_DEBUG_DIALOG // const RectDrawer = require( '@ckeditor/ckeditor5-utils/tests/_utils/rectdrawer' ).default;
+
+import cancelIcon from '@ckeditor/ckeditor5-core/theme/icons/cancel.svg';
+
+export enum DialogViewPosition {
```

Comment:
See my other concern about `enum` type.

Reply:
Done at [ck/14973-dialogs-docs](https://github.com/ckeditor/ckeditor5/tree/ck/14973-dialogs-docs) in a7448f1 - we ended up with the using `object as const`.